build/root/BUILD.root

```diff
@@ -65,6 +65,16 @@ filegroup(
         "//examples:all-srcs",
         "//federation:all-srcs",
         "//hack:all-srcs",
+        "//kubernetes/cluster:all-srcs",
+        "//kubernetes/docs:all-srcs",
+        "//kubernetes/examples:all-srcs",
+        "//kubernetes/federation/cluster:all-srcs",
+        "//kubernetes/federation/develop:all-srcs",
+        "//kubernetes/hack:all-srcs",
+        "//kubernetes/test/e2e/testing-manifests:all-srcs",
+        "//kubernetes/test/images:all-srcs",
+        "//kubernetes/test/kubemark:all-srcs",
+        "//kubernetes/third_party/htpasswd:all-srcs",
```

Comment:
I think maybe these aren't supposed to be here or update-bazel needs to be run, bazel-test will fail with these here and no build files

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/53561/pull-kubernetes-bazel-test/11273/